<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvOpDialog</class>
 <widget class="QDialog" name="AdvOpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>16</y>
     <width>128</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>高级选项</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>80</y>
     <width>320</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1. 是否使用容器和虚拟环境技术</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>100</y>
     <width>432</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>   例如docker、devcontainers、virtualenv等</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton1">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>144</y>
     <width>256</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>不要使用</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton2">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>176</y>
     <width>256</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>可用可不用</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton3">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>208</y>
     <width>256</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>建议使用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_yes">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>256</y>
     <width>96</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_no">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>256</y>
     <width>96</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
